ecosystem/resources/members/grover_controlled_diffuser.toml

```diff
@@ -0,0 +1,12 @@
+name = "Grover controlled-diffuser (CUs)"
+url = "https://github.com/albayaty/grover_controlled_diffuser"
+description = "The CUs is a new quantum diffusion operator for Grover’s algorithm to search for all correct solutions for Boolean oracles."
+licence = "Apache License 2.0"
+contact_info = "albayaty@pdx.edu"
+alternatives = "There are no alternatives to this project."
+affiliations = "The Electrical and Computer Eng. Dept. at Portland State University."
```

Comment:
Department of Electrical and Computer Engineering at Portland State University.

Reply:
Thank you!